frontend/src/components/Utils/time.js

```diff
@@ -93,15 +94,28 @@ export function getDateBarebones(msTimestamp, timezone=null) {
 }
 
 /**
- * Get a time in 24-hour ('HH:mm') form. 
+ * Get a time in 24-hour ('HH:mm') format. 
  * 
  * @param {string} msTimestamp Timestamp, in milliseconds since epoch.
  * @param {string} timezone The timezone which the string should be returned in.
- * @returns {string} The time in 24-hour (HH:mm) form.   
+ * @returns {string} The time in 24-hour (HH:mm) format.   
  */
 export function get24hTime(msTimestamp, timezone=null) {
   if (timezone === null) {
     return moment.tz(parseFloat(msTimestamp), '').format('HH:mm');
   }
   return moment.tz(parseFloat(msTimestamp), timezone).format('HH:mm');
+}
+
+/**
+ * Get a Firebase Timestamp object for time.
+ *
+ * @param {string} time The time in 'HH:mm' format.
+ * @param {string} date The date in 'YYYY-MM-DD' format.
+ * @param {string} tz The timezone in which the date takes place.
+ * @returns {firestore.Timestamp} Firestore timestamp object at the same time. 
+ */
+export function getFirebaseTime(time, date, tz) {
+  const mtzDate = moment.tz(time + " " + date, "HH:mm YYYY-MM-DD", tz);
```

Comment:
Curious, where did you get this syntax from? I am not a big fan of momentjs's documentation layout.

Reply:
[here](https://momentjs.com/timezone/docs/#/using-timezones/parsing-in-zone/)